portal-frontend/src/app/features/applications/view-submission/view-application-submission.component.html

```diff
@@ -68,17 +38,42 @@ <h4 *ngIf="application">Application ID: {{ application.fileNumber }}</h4>
             <div class="header">
               <h2>Applicant Submission</h2>
               <div class="btns-wrapper">
-                <button
-                  *ngIf="application.canEdit || application.canReview"
-                  mat-stroked-button
-                  color="warn"
-                  (click)="onCancel(application.uuid)"
-                >
-                  Cancel Application
-                </button>
                 <button mat-flat-button color="accent" (click)="onDownloadSubmissionPdf(application.fileNumber)">
                   Download PDF
                 </button>
+                <button mat-flat-button color="primary" *ngIf="application && application.canEdit" 
+                  [routerLink]="'/application/' + application.fileNumber + '/edit'">
+                  <div class="continue-button-content">
+                    <span>Continue Application</span>
+                    <mat-icon>keyboard_double_arrow_right</mat-icon>
+                  </div>
+                </button>
+                <button (click)="onReview(application.fileNumber)"  mat-flat-button color="primary"
+                  *ngIf="
+                    application && application.canReview && (
+                    application.status.code === SUBMISSION_STATUS.IN_REVIEW_BY_LG || 
+                    application.status.code === SUBMISSION_STATUS.RETURNED_TO_LG || 
+                    application.status.code === SUBMISSION_STATUS.SUBMITTED_TO_LG)
+                  ">
+                  <div class="continue-button-content">
+                    <span
+                      *ngIf="
+                        application.status.code === SUBMISSION_STATUS.IN_REVIEW_BY_LG ||
+                        application.status.code === SUBMISSION_STATUS.RETURNED_TO_LG
+                      "
+                      (click)="onReview(application.fileNumber)"
+                      class="subheading2"
+                      >Continue Application Review</span
```

Comment:
Inverse of above: will "Continue Application" button be visible on application submission tab?

Reply:
yes thats the requirement, show on all pages

Reply:
It's a bit unclear, because it does not mention "continue application review" yet that is obviously needed, and the Figma never shows this case, but I belive you. As discussed before, this isn't the place for req discussion anyway. I just wanted to make sure this was intentional.